docs/API-functions.md

```diff
@@ -1018,116 +1018,8 @@ Reports the current statistics
-measured in TCP, which is the time between sending a data packet of a particular 
-sequence number and receiving `UMSG_ACK` with a sequence number that is later 
-by 1. Forward RTT isn't being measured or reported in SRT, although some
-research works have shown that these values, even though shuold be the same,
-happen to differ, that is, "reverse RTT" seems to be more optimistic.
-
-* `mbpsBandwidth`: The bandwidth in Mb/s. The bandwidth is measured at the 
-receiver, which sends back a running average calculation to the sender with
-the ACK message.
-
-* `byteAvailSndBuf`: The number of bytes available in the sender buffer. This 
-value decreases with data scheduled for sending by the application, and increases 
-with every ACK received from the receiver, after the packets are sent over 
-the UDP link.
-
-* ` byteAvailRcvBuf`: The number of bytes available in the receiver buffer.
-This value increases after the application extracts the data from the socket
-(uses one of `srt_recv*` functions) and decreases with every packet received
-from the sender over the UDP link.
-
-* `mbpsMaxBW`: The maximum bandwidth in Mb/s. Usually this is the setting from 
-the `SRTO_MAXBW` option, which may include the value 0 (unlimited). Under certain 
-conditions a nonzero value might be be provided by the appropriate congestion 
-control module, although none of the built-in congestion control modules 
-currently uses it.
-
-* `byteMSS`: Same as a value from `SRTO_MSS` option, "Message Segment Size".
-It's the size of the MTU unit (size of the UDP packet used for transport,
-including all possible headers, that is Ethernet, IP and UDP), default 1500.
-
-* `pktSndBuf`: The number of packets in the send buffer that are already 
-scheduled for sending or even possibly sent, but not yet acknowledged.
-
-* `byteSndBuf`: Same as `pktSndBuf`, in bytes.
-
-* `msSndBuf`: Same as `pktSndBuf`, but expressed as a time interval between the
-oldest and the latest packet scheduled for sending.
-
-* `msSndTsbPdDelay`: If `SRTO_TSBPDMODE` is on (default for **live mode**), it 
-returns the value of `SRTO_PEERLATENCY`, otherwise 0.
-
-* `pktRcvBuf`: Number of packets in the receiver buffer. Note that in **live mode** 
-(with `SRTO_TSBPDMODE` turned on, default) some packets must stay in the buffer 
-and will not be signed off to the application until the "time to play" comes. 
-In **file mode** (both stream and message) it means that all that is above 0 can
-(and shall) be read right now.
-
-* `byteRcvBuf`: Like `pktRcvBuf`, in bytes.
-
-* `msRcvBuf`: Time interval between the first and last available packets in the
-receiver buffer. Note that this range includes all packets regardless of whether 
-they are ready to play or not (regarding the **live mode**)..
-
-* `msRcvTsbPdDelay`: If `SRTO_TSBPDMODE` is on (default for **live mode**), it 
-returns the value of `SRTO_RCVLATENCY`; otherwise 0.
+`SRT_TRACEBSTATS` is an alias to `struct CBytePerfMon`. For complete description
```

Comment:
`SRT_TRACEBSTATS` is an alias to `struct CBytePerfMon`. For a complete description